WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsFragment.kt

```diff
@@ -153,8 +153,19 @@ class StatsFragment : DaggerFragment() {
         })
 
         viewModel.toolbarHasShadow.observe(this, Observer { hasShadow ->
-            val elevation = if (hasShadow == true) resources.getDimension(R.dimen.appbar_elevation) else 0f
-            app_bar_layout.postDelayed({ ViewCompat.setElevation(app_bar_layout, elevation) }, 100)
+            app_bar_layout.postDelayed(
+                    {
+                        if (app_bar_layout != null) {
+                            val elevation = if (hasShadow == true) {
+                                resources.getDimension(R.dimen.appbar_elevation)
+                            } else {
+                                0f
+                            }
```

Comment:
@planarvoid I had to resolve some conflicts with `develop` for your stats change. Can you confirm this looks okay? In particular, I made the assignment span multiple lines to fix a lint error.

Reply:
looks good to me 👍  thanks for pinging me